paasta_tools/instance/kubernetes.py

```diff
@@ -618,20 +618,33 @@ async def kubernetes_status_v2(
         job_config.get_instances() if desired_state != "stop" else 0
     )
     status["bounce_method"] = job_config.get_bounce_method()
-    status["versions"] = versions_task.result()
+
+    try:
+        pods_task.result()  # just verifies we have a valid result
```

Comment:
so, to clarify, this is so that the pods_task will return its exception here rather than deeper down in the stack where we don't have the error handling, yeah?